sql-streaming-sqs/README.md

```diff
@@ -63,6 +63,18 @@ shouldSortFiles|true|whether to sort files based on timestamp while listing them
 useInstanceProfileCredentials|false|Whether to use EC2 instance profile credentials for connecting to Amazon SQS
 maxFilesPerTrigger|no default value|maximum number of files to process in a microbatch
 maxFileAge|7d|Maximum age of a file that can be found in this directory
+messageWrapper|None| - 'None' if SQS contains plain S3 message. <br/> - 'SNS' if SQS contains S3 notification message which came from SNS. <br/> Please see 'SQS vs SNS' section for more details 
+
+## SQS vs SNS
```

Comment:
Maybe change the title to Using Multiple Consumers

Reply:
makes total sense. Changed, thanks